This patch updates the versions to current the ones (as
of 2020-09-01), and removes some of the hardcoded versions
to be able to use newer versions later on.

--- autogen.sh.orig
+++ autogen.sh
@@ -8,16 +8,16 @@
 # tools and you shouldn't use this script.  Just call ./configure
 # directly.
 
-ACLOCAL=${ACLOCAL-aclocal-1.9}
+ACLOCAL=${ACLOCAL-aclocal-1.16}
 AUTOCONF=${AUTOCONF-autoconf}
 AUTOHEADER=${AUTOHEADER-autoheader}
-AUTOMAKE=${AUTOMAKE-automake-1.9}
+AUTOMAKE=${AUTOMAKE-automake-1.16}
 LIBTOOLIZE=${LIBTOOLIZE-libtoolize}
 
-AUTOCONF_REQUIRED_VERSION=2.54
-AUTOMAKE_REQUIRED_VERSION=1.9.6
-INTLTOOL_REQUIRED_VERSION=0.36.3
-LIBTOOL_REQUIRED_VERSION=1.5
+AUTOCONF_REQUIRED_VERSION=2.69
+AUTOMAKE_REQUIRED_VERSION=1.16.2
+INTLTOOL_REQUIRED_VERSION=0.51.0
+LIBTOOL_REQUIRED_VERSION=2.4.6
 
 
 PROJECT="GIMP Save for Web plug-in"
@@ -101,27 +101,27 @@ echo -n "checking for automake >= $AUTOMAKE_REQUIRED_VERSION ... "
 if ($AUTOMAKE --version) < /dev/null > /dev/null 2>&1; then
    AUTOMAKE=$AUTOMAKE
    ACLOCAL=$ACLOCAL
-elif (automake-1.15 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.15
-   ACLOCAL=aclocal-1.15
-elif (automake-1.14 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.14
-   ACLOCAL=aclocal-1.14
-elif (automake-1.13 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.13
-   ACLOCAL=aclocal-1.13
-elif (automake-1.12 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.12
-   ACLOCAL=aclocal-1.12
-elif (automake-1.11 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.11
-   ACLOCAL=aclocal-1.11
-elif (automake-1.10 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.10
-   ACLOCAL=aclocal-1.10
-elif (automake-1.9 --version) < /dev/null > /dev/null 2>&1; then
-   AUTOMAKE=automake-1.9
-   ACLOCAL=aclocal-1.9
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
+elif (automake --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake
+   ACLOCAL=aclocal
 else
     echo
     echo "  You must have automake $AUTOMAKE_REQUIRED_VERSION or newer installed to compile $PROJECT."
@@ -170,7 +170,7 @@ test $TEST_TYPE $FILE || {
 echo
 echo "I am going to run ./configure with the following arguments:"
 echo
-echo "  --enable-maintainer-mode $AUTOGEN_CONFIGURE_ARGS $@"
+echo "  $AUTOGEN_CONFIGURE_ARGS $@"
 echo
 
 if test -z "$*"; then
@@ -219,7 +219,7 @@ intltoolize --automake || exit 1
 
 cd $ORIGDIR
 
-$srcdir/configure --enable-maintainer-mode $AUTOGEN_CONFIGURE_ARGS "$@"
+$srcdir/configure $AUTOGEN_CONFIGURE_ARGS "$@"
 RC=$?
 if test $RC -ne 0; then
   echo
